### ISSUE TITLE
Test Zincsearch

### ISSUE THREAD

**Issue by djklim87** [db-benchmarks/db-benchmarks#44]

**Comment by djklim87** [comment]
Zinc caches queries and seems there is no way to disable it 
https://github.com/zincsearch/zincsearch/issues/351

**Comment by djklim87** [comment]
I asked them to clarification https://github.com/zincsearch/zincsearch/issues/978

**Comment by djklim87** [comment]
Blocked by https://github.com/zincsearch/zincsearch/issues/980

**Comment by marclaporte** [comment]
Is it worth it?

"We have no energy on this project now. if anyone want to maintain we will welcome."
https://github.com/zincsearch/zincsearch/issues/782

**Comment by sanikolaev** [comment]
> Is it worth it?

Good point, thanks @marclaporte. If the project is being discontinued, it might not make sense to maintain the tests. From what I understand, they are now focusing on https://github.com/openobserve/openobserve as the successor to ZincSearch.

@djklim87, please check how similar they are and see if we can modify the tests with minimal changes to benchmark Openobserve instead.

**Comment by djklim87** [comment]
We decided to wait few weeks until OpenObserve implement normal Logstash support
